index.html

```diff
@@ -357,7 +357,7 @@ <h2><dfn>SmartCardConnection</dfn> interface</h2>
           Promise&lt;undefined&gt; setAttribute([EnforceRange] unsigned long tag, BufferSource value);
         };
 
-        callback TransactionStartedCallback = Promise&lt;any&gt; ();
+        callback SmartCardTransactionCallback = Promise&lt;SmartCardDisposition&gt; ();
```

Comment:
I think this would technically be a `SmartCardDisposition?` to represent the option of not returning a disposition.

Reply:
Originally my thought was that the callback **had to** return a disposition. But returning nothing as an equivalent of `"leave"` is a nice convenience. Done.